scripts/check-v-next-npm-scripts.js

```diff
@@ -0,0 +1,56 @@
+// @ts-check
+const fs = require("fs");
+const path = require("path");
+
+const templatePackageJson = require("../v-next/template-package/package.json");
+
+const vNextDir = path.resolve(__dirname, "../v-next");
+const dirs = fs.readdirSync(vNextDir, { withFileTypes: true });
+
+let errorsFound = false;
+
+for (const dir of dirs) {
+  if (!dir.isDirectory()) {
+    continue;
+  }
+
+  const packageJsonPath = path.resolve(vNextDir, dir.name, "package.json");
+  const packageJson = require(packageJsonPath);
+
+  for (const scriptName in templatePackageJson.scripts) {
+    if (scriptName === "clean") {
+      if (
+        !packageJson.scripts[scriptName].startsWith(
+          templatePackageJson.scripts[scriptName]
+        )
+      ) {
+        console.error(`Mismatch in script ${scriptName} in ${dir.name}`);
```

Comment:
do you think we could use node assert here?

Reply:
that leads to a single error being reported instead of all of them